masonite/middlewares/CsrfMiddleware.py

```diff
@@ -0,0 +1,36 @@
+from masonite.exceptions import InvalidCSRFToken
+
+
+class CsrfMiddleware(object):
+    """
+    Verify csrf token middleware
+    """
+
+    exempt = []
+
+    def __init__(self, Request, CSRF):
+        self.request = Request
+        self.csrf = CSRF
+
+    def before(self):
+        if self.request.is_post():
+            token = self.request.input('csrftoken')
```

Comment:
Notice that you're getting the `.input('csrftoken')` but inside the input on line 19 of `CsrfProvider` it is `csrf_token`. I'd go with the underscore one